osutil/cp.go

```diff
@@ -127,23 +137,32 @@ func runCpPreserveAll(path, dest string) error {
 	return nil
 }
 
+func runSync() error {
+	return runCmd(exec.Command("sync"), "sync")
```

Comment:
that is: I could walk the tree that's just been copied and call sync on everything i find, but it'll be a lot of throwaway work (hoping to drop having to use cp/sync at some point)